lndb/_check_versions.py

```diff
@@ -1,5 +1,5 @@
 from lnhub_rest import __version__ as lnhub_rest_v
 from packaging import version
 
-if version.parse(lnhub_rest_v) != version.parse("0.8.2"):
+if version.parse(lnhub_rest_v) != version.parse("0.9.2"):
     raise SystemExit("Please upgrade lnhub_rest: pip install lnhub_rest==0.8.2")
```

Comment:
Fred, can you also update the error message, here?

Reply:
Users are going to be confused otherwise :D

Reply:
Sorry I miss this